entity-store/src/main/kotlin/jetbrains/exodus/entitystore/orientdb/query/OCondition.kt

```diff
@@ -117,11 +117,21 @@ class OEdgeExistsCondition(
     }
 }
 
+class OSingleVertexCondition(
```

Comment:
I would better call it like this to highlight what is actually in the condition
```suggestion
class OIdEqualCondition(
```